### DISCUSSION TITLE
The examples seem broken

### DISCUSSION THREAD

**Discussion by sainikhileshmurtyk** [DylanMMarques/Jolab.jl#21] [Q&A] [answered]
Hello,

Thank you for your work on this wonderful project. I had used the package some time ago and it had worked flawlessly. However, when I tried it out again on a fresh install, the examples don't seem to run. I think the issue could be with changes to the method names in the newer version. However, because of inadequate documentation, the task becomes even harder. Just wanted to bring this to your attention.

Best,

**Comment by DylanMMarques** [comment, marked as answer]
Hello,

Thanks for your message. I have created a new released with the examples updated. I apolize for the outdated examples but latelly I couldn't spend as much time as I would like in Jolab. However, in the next months, I will be able to spend more time working on Jolab. I will start by focusing on documenting the code which is currently very very limited. 

If you have developed scripts you should be able to use them in the newer version of Jolab if you update your script according to the release notes (https://github.com/DylanMMarques/Jolab.jl/releases/tag/v0.1.5). Otherwise you can also install an old version of Jolab with:
``` julia
] add Jolab@0.1.X
```
where X is the version you were using.

The current documentation is really limited so feel free to e-mail me if you have any question or if you any feedback about Jolab.

**Comment by sainikhileshmurtyk** [comment]
Hello,

Thank you so much for your quick reply. I'll update the scripts as you mentioned. 

Best
